apps/src/blocksCommon.js

```diff
@@ -23,18 +30,27 @@ exports.install = function (blockly, blockInstallOptions) {
 
 function installControlsRepeatSimplified(blockly, skin) {
   // Re-uses the repeat block generator from core
-  blockly.JavaScript.controls_repeat_simplified =
-    blockly.JavaScript.controls_repeat;
-  blockly.JavaScript.controls_repeat_simplified_dropdown =
-    blockly.JavaScript.controls_repeat;
+  blockly.customBlocks.copyBlockGenerator(
```

Comment:
Did you add this util mainly so you don't have to write out
`blockly.JavaScript.forBlock.controls_repeat_simplified = blockly.JavaScript.forBlock.controls_repeat`?

Reply:
Only partially. If I had changed that directly this would crash in any CDO Blockly lab since forBlock is undefined.